lib/file-list.js

```diff
@@ -114,6 +114,7 @@ class FileList {
     return lastCompletedRefresh
   }
 
+  // TODO - Change this so that it's not a getter. Getters shouldn't do lots of processing!
```

Comment:
Nit: `TODO:` instead of `TODO -` :)

Reply:
Hi - I really don't want to spend the 15 minutes on context switching to remove a '-'. Can this be accepted as is (assuming no other issues)?